flyteplugins/go.mod

```diff
@@ -14,11 +14,10 @@ require (
 	github.com/flyteorg/flyte/flytepropeller v0.0.0-00010101000000-000000000000
 	github.com/flyteorg/flyte/flytestdlib v0.0.0-00010101000000-000000000000
 	github.com/go-test/deep v1.0.7
-	github.com/golang/protobuf v1.5.3
+	github.com/golang/protobuf v1.5.4
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/imdario/mergo v0.3.13
-	github.com/kubeflow/common v0.4.3
-	github.com/kubeflow/training-operator v1.5.0-rc.0
+	github.com/kubeflow/training-operator v1.8.0
```

Comment:
One important change here is that `github.com/kubeflow/common` isn't needed anymore as it has been integrated into `github.com/kubeflow/training-operator`.